motion/lib/src/input_stream.dart

```diff
@@ -26,44 +26,53 @@ class InputStream extends StatefulWidget {
 }
 
 class InputStreamState extends State<InputStream> {
-  late Stream<MotionEvent> inputStream;
+  Stream<MotionEvent>? inputStream;
 
   DateTime? lastPointerEventTime;
 
   @override
   void initState() {
     super.initState();
+    initialiseInputStream();
+  }
+
+  void initialiseInputStream() async {
+    await Motion.instance.initialize();
+    Motion.instance.setUpdateInterval(60.fps);
```

Comment:
Some users may need to delay the plugin's initialization, for example if a permission request is needed, to show a rationale before requesting the permission. 

More generally, I believe auto-initializing with quite arbitrary values may cause more problem than it will solve.

Reply:
I agree with your comment and this can be fixed in this PR.